Gruntfile.js

```diff
@@ -42,7 +42,7 @@ module.exports = function (grunt) {
             '<%= dirs.public.css %>/autofill.css': ['<%= ddgAutofill %>/autofill.css']
         },
         unitTest: {
-            '<%= dirs.unitTest.build %>/background.js': ['<%= dirs.unitTest.background %>/**/*.js', '!<%= dirs.unitTest.background %>/reference-tests/**/*.js'],
```

Comment:
we no longer need to exclude reference-tests repo, because it's not in the tests folder